packages/adapters-library/package.json

```diff
@@ -32,7 +32,7 @@
     "test:detect-errors": "jest src/tests/detect-errors.test.ts",
     "test:update:adapter-snapshot": "jest src/scripts/adapterBuilder/newAdapterCommand.test.ts -u",
     "test:tsc": "tsc --noEmit -p tsconfig.json",
-    "test:integration": "DEFI_ADAPTERS_TEST_FILTER_PROTOCOL=$npm_config_protocol jest --coverage --coverageDirectory='coverage/e2e/' src/adapters/integration.test.ts",
+    "test:integration": "DEFI_ADAPTERS_TEST_FILTER_PROTOCOL=$npm_config_protocol DEFI_ADAPTERS_TEST_FILTER_PRODUCT=$npm_config_productid jest --coverage --coverageDirectory='coverage/e2e/' src/adapters/integration.test.ts",
```

Comment:
Apparently, --product is a reserved parameter, because using `$npm_config_product` and passing `--product xxx` would not work. However, changing it to `productid` worked, so I did the same for protocol for consistency.